features/bootstrap/authentication.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Authenticating via API
+In order to authenticate
+as an API client
+I need to be able to register and login into system
+
+Scenario: Registering new user
```

Comment:
You can tag your suites and scenarios:
```
@api @user @success
Scenario: Registering new user
```

It could be really useful in case you'd want to check only user-related things. It's also superuseful with using tags like `@current` when you want to run only selected tests during debugging.